Commutation.py

```diff
@@ -4975,9 +4958,66 @@ def __init__(self):
         self.ui = Ui_MainWindow()
         self.ui.setupUi(self)
         self.ui.pushButton.clicked.connect(self.btnClicked)
+        self.ui.pushButton_1.clicked.connect(self.btn_1Clicked)
+
+    """
+    v   Получение логов при ошибке
+    """
+
+    def log_uncaught_exceptions(ex_cls, ex, tb):
+        text = '{}: {}:\n'.format(ex_cls.__name__, ex)
+        import traceback
+        text += ''.join(traceback.format_tb(tb))
+
+        print(text)
+        quit()
+
+    import sys
+    sys.excepthook = log_uncaught_exceptions
+
+    """
+    ^
+    """
+
+    def btn_1Clicked(self):
+        self.ui.text += 'Запрос ответа от PV-4202...\n'
```

Comment:
Здесь ты выводишь надписи состояния только для блока 4202, а есть еще 4201. Смотри код внимательнее
```suggestion
        self.ui.text += 'Запрос ответа от PV-4201...\n'
        self.ui.text += 'Запрос ответа от PV-4202...\n'
        и тд
```